source/includes/gcp/_instance_groups.md

```diff
@@ -1 +1,225 @@
+
+#### Retrieve an instance
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/gcp/test-area/instancegroups/2986056884972096897"
+
+# The above command returns JSON structured like this:
+```
+```json
+{
+  "data": {
+    "creationTimestamp": "2019-09-06T12:28:47.389-07:00",
+    "description": "",
+    "fingerprint": "42WmSpB8rSM=",
+    "kind": "compute#instanceGroup",
+    "namedPorts": [
+      {
+        "name": "http",
+        "port": 80
+      }
+    ],
+    "network": "https://www.googleapis.com/compute/v1/projects/test-area/global/networks/default",
+    "selfLink": "https://www.googleapis.com/compute/v1/projects/test-area/zones/us-central1-a/instanceGroups/ig-test-vmd",
+    "size": 0,
+    "subnetwork": "https://www.googleapis.com/compute/v1/projects/test-area/regions/us-central1/subnetworks/default",
+    "zone": "https://www.googleapis.com/compute/v1/projects/test-area/zones/us-central1-a",
+    "id": "2986056884972096897",
+    "name": "ig-test-vmd",
+    "shortZone": "us-central1-a",
+    "shortRegion": "us-central1"
+  }
+}
+```
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/instancegroups/:id</code>
+
+Retrieve an instance group in a given [environment](#administration-environments)
+
+Attributes | &nbsp;
+------- | -----------
+`creationTimestamp`<br/>*string* | Creation timestamp in RFC3339 text format
+`description`<br/>*string* | An optional description of this resource
+`fingerprint`<br/>*string* | A short sequence of bytes used to identify the named ports
+`kind`<br/>*string* | Type of the resource
+`namedPorts`<br/>*object* | Assigns a name to a port number. This allows the system to reference ports by the assigned name instead of a port number. Named ports apply to all instances in this instance group.
+`network`<br/>*string* | The network to which the instance group is connected to
+`selfLink`<br/>*string* | Server-defined URL for this resource
+`size`<br/>*integer* | The number of instances in the group
+`subnetwork`<br/>*string* | The sub-network within the network the instance group is connected to
+`zone`<br/>*string* | URL of the zone where the instance group resides
+`id`<br/>*UUID* | The id of the instance group
+`name`<br/>*string* | The display name of the instance group
+`shortZone`<br/>*string* | A short version of the zone name
+`shortRegion`<br/>*string* | A short version of the region name
+
+<!-------------------- CREATE AN INSTANCE GROUP -------------------->
+
+#### Create an instance group
+
+```shell
+curl -X POST \
+   -H "Content-Type: application/json" \
+   -H "MC-Api-Key: your_api_key" \
+   -d "request_body" \
+   "https://cloudmc_endpoint/v1/services/gcp/test-area/instancegroups"
+
+# Request example:
+```
+```json
+{
+	"name": "my-group",
+	"shortRegion": "northamerica-northeast1",
```

Comment:
Is it necessary to provide the shortRegion? We could infer it since we already have the shortZone.

Reply:
I'll check

Reply:
Any findings?

Reply:
It's doable. I'll make sure we do it.

Reply:
Done